util/verible-style-lint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# this script runs the verible style linte on all system verilog
+# files under hw/{ip,vendor,top_earlgrey}
+#
+# TODO: integrate this with Fusesoc and the other linting flows.
+
+REPORT_FILE="verible-style-lint.rpt"
+NUM_PROCS=8
+
+# get all system verilog files and pipe through style linter
+find hw/{ip,vendor,top_earlgrey} -type f -name "*.sv" -o -name "*.svh" |  \
+    xargs -n 1 -P $NUM_PROCS /tools/verible/verilog_lint                  \
```

Comment:
should we add `--rules=-macro-name-style` to exclude this rule which DV doesn't follow

Reply:
we can do that for now, but this needs some more discussion at some point. 

ideal would be if we had a way to configure the tool using something like a policy file, plus support for waiver files (as opposed to inline waivers)...

do you have a list of specific rules?

Reply:
`--rules=-macro-name-style` is the only waiver I use now.

Reply:
ok done

Reply:
The ABSL flags library supports a generic `--flagfile=FILE` option where you can store command-line flags, if you prefer.  This is available on all Verible command-line tools: verilog_lint, verilog_format, verilog_syntax.

Reply:
Ok, good to know, thanks. I may use that in a follow-up commit if we have to waive more rules.